src/Compilers/Test/Core/CommonTestBase.cs

```diff
@@ -37,6 +39,27 @@ public enum Verification
         PassesOrFailFast = 1 << 4,
     }
 
+    public readonly struct Verification
+    {
+        public VerificationStatus Status { get; init; }
```

Comment:
`required`?

Reply:
Not really required. `default(Verification)` is intentionally equivalent to `Pass`.